### ISSUE TITLE
how to generate TFRecords. 

### ISSUE THREAD

**Issue by Abduoit** [datitran/raccoon_dataset#10]
I used raccoon images and annotations in order to generate TFRecords. 
I used this [link](https://github.com/tensorflow/models/blob/master/object_detection/g3doc/preparing_inputs.md#generating-the-pascal-voc-tfrecord-files)
and I change this [line](https://github.com/tensorflow/models/blob/master/object_detection/create_pascal_tf_record.py#L162)

But the generating TFRecords step is wrong. I get this error. Any help would be grateful

```
(abdu-py2) jesse@jesse-System-Product-Name:~/abdu-py2/models$ python object_detection/create_pascal_tf_record.py     --label_map_path=object_detection/data/pascal_label_map.pbtxt     --data_dir=object_detection/VOCdevkit --year=VOC2012 --set=train     --output_path=object_detection/data/newpascal_train.record
/home/jesse/abdu-py2/models/object_detection/utils/dataset_util.py:75: FutureWarning: The behavior of this method will change in future versions. Use specific 'len(elem)' or 'elem is not None' test instead.
  if not xml:
Traceback (most recent call last):
  File "object_detection/create_pascal_tf_record.py", line 183, in <module>
    tf.app.run()
  File "/home/jesse/abdu-py2/local/lib/python2.7/site-packages/tensorflow/python/platform/app.py", line 48, in run
    _sys.exit(main(_sys.argv[:1] + flags_passthrough))
  File "object_detection/create_pascal_tf_record.py", line 176, in main
    FLAGS.ignore_difficult_instances)
  File "object_detection/create_pascal_tf_record.py", line 87, in dict_to_tf_example
    encoded_jpg = fid.read()
  File "/home/jesse/abdu-py2/local/lib/python2.7/site-packages/tensorflow/python/lib/io/file_io.py", line 118, in read
    self._preread_check()
  File "/home/jesse/abdu-py2/local/lib/python2.7/site-packages/tensorflow/python/lib/io/file_io.py", line 78, in _preread_check
    compat.as_bytes(self.__name), 1024 * 512, status)
  File "/usr/lib/python2.7/contextlib.py", line 24, in __exit__
    self.gen.next()
  File "/home/jesse/abdu-py2/local/lib/python2.7/site-packages/tensorflow/python/framework/errors_impl.py", line 466, in raise_exception_on_not_ok_status
    pywrap_tensorflow.TF_GetCode(status))
tensorflow.python.framework.errors_impl.NotFoundError: object_detection/VOCdevkit/images/JPEGImages/raccoon-1.png
```

**Comment by datitran** [comment]
Well it seems that your path is not correct: `tensorflow.python.framework.errors_impl.NotFoundError: object_detection/VOCdevkit/images/JPEGImages/raccoon-1.png`.

**Comment by Abduoit** [comment]
In order to check the image's path, I used `vim classname.xml` and I found out that the file `.xml` has wrong path, although I placed the annotations and images in correct directory. For some reasons the files `.xml` that I used have wrong paths. Any way, I had to re-annotate the images again in order to get different path, and I successfully re-generated TFRecords files.

Thx for your help, plz keep on doing amazing work.
